server/server/models/comment.py

```diff
@@ -0,0 +1,68 @@
+class Comment:
+    @staticmethod
+    def get_for_event(request, event_id):
+        obj = request.mongo.comments.find_one({
+            'event_id': event_id
+        })
+        if obj is None:
+            return {'comments': []}
+        return obj['comments']
+
+    @staticmethod
+    def add_comment(request, event_id, author_id, author_nickname, comment_msg,
+                    timestamp, father_comment_id):
+        new_comment = {
+            'author_id': author_id,
+            'author_nickname': author_nickname,
+            'timestamp': timestamp,
```

Comment:
It's a good place to generate timestamp, and this should be done using MongoDB functions instead of Python ones.

Reply:
Is it very important to use MongoDB function instead of Python? I need to add a new package for this, but with Python, it's easier.

Reply:
Ok, you can use Python's stuff.